### PR TITLE
Added Newman custom reporter docs

### PR DESCRIPTION
**Pull request by kunagpal**
Includes details on:

* Creating custom reporters 
* Using custom reporters

cc @czardoz @godfrzero

### CODE REVIEW

**Comment by godfrzero** [comment]
Looks good to me as far as syntax and whatnot goes. @czardoz can comment on the technical accuracy.

File: _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md [hunk 1]
```diff
@@ -217,6 +217,59 @@ newman.run({
...
+```
+
+Scoped reporter package names like `@myorg/newman-reporter-<name>` are also supported. Working reporter examples can be found in https://github.com/postmanlabs/newman/tree/develop/lib/reporters.
+
+#### Using custom reporters
+In order to use the custom reporter, it will have to be installed first. For instance, to use the [Newman teamcity reporter](https://www.npmjs.com/package/newman-reporter-teamcity):
+
+Install the reporter package. 
+```
+npm i -S newman-reporter-teamcity
```
Comment (czardoz) [inline _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md]: Should not use `-S`, most of the users who are going to _use_ custom reporters are not Node.js programmers, they just need the reporter installed.

File: _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md [hunk 2]
```diff
@@ -217,6 +217,59 @@ newman.run({
 });
 ```
 
+### Custom reporters
+
+#### Building custom reporters
```
Comment (czardoz) [inline _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md]: Need to document exact steps to do this. Refer to https://github.com/postmanlabs/newman/issues/986#issuecomment-293505135

Essentially, we need to enable a non-nodejs programmer to quickly get a reporter up & running.

Comment (kunagpal) [inline _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md]: More details have been added.

File: _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md (hunk 1 above)
Comment (kunagpal) [inline _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md]: Has been corrected.

File: _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md (hunk 2 above)
Comment (kunagpal) [inline _posts/06_api_testing_and_collection_runner/2016-07-11-newman_intro.md]: More details have been added.

**Comment by czardoz** [comment]
@godfrzero Ready for merging!

**Pull request merged**